<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1164</width>
    <height>736</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Radar</string>
  </property>
  <property name="windowIcon">
   <iconset resource="res.qrc">
    <normaloff>:/icons/res/Icon1.png</normaloff>:/icons/res/Icon1.png</iconset>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QMdiArea" name="mdiArea"/>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1164</width>
     <height>25</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>Файл</string>
    </property>
    <addaction name="action1"/>
    <addaction name="action39"/>
    <addaction name="separator"/>
    <addaction name="action_2"/>
   </widget>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string>Редактировать</string>
    </property>
   </widget>
   <widget class="QMenu" name="menu_2">
    <property name="title">
     <string>Поиск</string>
    </property>
    <addaction name="action_4"/>
    <addaction name="action_7"/>
    <addaction name="separator"/>
    <addaction name="action_8"/>
   </widget>
   <widget class="QMenu" name="menu_3">
    <property name="title">
     <string>Взять</string>
    </property>
   </widget>
   <widget class="QMenu" name="menu_4">
    <property name="title">
     <string>Показать</string>
    </property>
   </widget>
   <widget class="QMenu" name="menu_5">
    <property name="title">
     <string>Настройки</string>
    </property>
   </widget>
   <widget class="QMenu" name="menu_6">
    <property name="title">
     <string>Окно</string>
    </property>
    <addaction name="action"/>
    <addaction name="action_3"/>
    <addaction name="separator"/>
    <addaction name="action_5"/>
    <addaction name="action_6"/>
    <addaction name="separator"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menu"/>
   <addaction name="menu_2"/>
   <addaction name="menu_3"/>
   <addaction name="menu_4"/>
   <addaction name="menu_5"/>
   <addaction name="menu_6"/>
  </widget>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="action1"/>
   <addaction name="action2"/>
   <addaction name="action3"/>
   <addaction name="action4"/>
   <addaction name="action5"/>
   <addaction name="action6"/>
   <addaction name="action7"/>
   <addaction name="separator"/>
   <addaction name="action8"/>
   <addaction name="action9"/>
   <addaction name="action10"/>
   <addaction name="action11"/>
   <addaction name="action12"/>
   <addaction name="separator"/>
   <addaction name="action13"/>
   <addaction name="action14"/>
   <addaction name="separator"/>
   <addaction name="action15"/>
   <addaction name="action16"/>
   <addaction name="action17"/>
   <addaction name="action18"/>
   <addaction name="action19"/>
   <addaction name="separator"/>
   <addaction name="action20"/>
   <addaction name="action21"/>
   <addaction name="action22"/>
   <addaction name="separator"/>
   <addaction name="action23"/>
   <addaction name="action24"/>
   <addaction name="separator"/>
   <addaction name="action25"/>
   <addaction name="action26"/>
   <addaction name="separator"/>
   <addaction name="action27"/>
  </widget>
  <widget class="QToolBar" name="toolBar_2">
   <property name="windowTitle">
    <string>toolBar_2</string>
   </property>
   <attribute name="toolBarArea">
    <enum>LeftToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="action29"/>
   <addaction name="action30"/>
   <addaction name="action31"/>
   <addaction name="action32"/>
   <addaction name="action33"/>
   <addaction name="action34"/>
   <addaction name="action35"/>
   <addaction name="action36"/>
   <addaction name="action37"/>
   <addaction name="action38"/>
   <addaction name="separator"/>
   <addaction name="action39"/>
   <addaction name="action40"/>
   <addaction name="action41"/>
   <addaction name="action42"/>
   <addaction name="action43"/>
   <addaction name="action44"/>
  </widget>
  <action name="action_2">
   <property name="text">
    <string>Выход</string>
   </property>
  </action>
  <action name="action1">
   <property name="checkable">
    <bool>false</bool>
   </property>
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/icons/res/Bitmap7.bmp</normaloff>:/icons/res/Bitmap7.bmp</iconset>
   </property>
   <property name="text">
    <string>Открыть реперторий</string>
   </property>
   <property name="iconText">
    <string>Открыть реперторий</string>
   </property>
   <property name="toolTip">
    <string>Открыть реперторий</string>
   </property>
   <property name="visible">
    <bool>true</bool>
   </property>
   <property name="iconVisibleInMenu">
    <bool>false</bool>
   </property>
  </action>
  <action name="action2">
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/icons/res/Bitmap2.bmp</normaloff>:/icons/res/Bitmap2.bmp</iconset>
   </property>
   <property name="text">
    <string>2</string>
   </property>
  </action>
  <action name="action3">
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/icons/res/Bitmap3.bmp</normaloff>:/icons/res/Bitmap3.bmp</iconset>
   </property>
   <property name="text">
    <string>3</string>
   </property>
  </action>
  <action name="action4">
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/icons/res/Bitmap4.bmp</normaloff>:/icons/res/Bitmap4.bmp</iconset>
   </property>
   <property name="text">
    <string>4</string>
   </property>
  </action>
  <action name="action5">
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/icons/res/Bitmap5.bmp</normaloff>:/icons/res/Bitmap5.bmp</iconset>
   </property>
   <property name="text">
    <string>5</string>
   </property>
  </action>
  <action name="action6">
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/icons/res/Bitmap42.bmp</normaloff>:/icons/res/Bitmap42.bmp</iconset>
   </property>
   <property name="text">
    <string>6</string>
   </property>
  </action>
  <action name="action7">
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/icons/res/Bitmap6.bmp</normaloff>:/icons/res/Bitmap6.bmp</iconset>
   </property>
   <property name="text">
    <string>7</string>
   </property>
  </action>
  <action name="action8">
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/icons/res/Bitmap7.bmp</normaloff>:/icons/res/Bitmap7.bmp</iconset>
   </property>
   <property name="text">
    <string>8</string>
   </property>
  </action>
  <action name="action9">
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/icons/res/Bitmap8.bmp</normaloff>:/icons/res/Bitmap8.bmp</iconset>
   </property>
   <property name="text">
    <string>9</string>
   </property>
  </action>
  <action name="action10">
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/icons/res/Bitmap9.bmp</normaloff>:/icons/res/Bitmap9.bmp</iconset>
   </property>
   <property name="text">
    <string>10</string>
   </property>
  </action>
  <action name="action11">
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/icons/res/Bitmap10.bmp</normaloff>:/icons/res/Bitmap10.bmp</iconset>
   </property>
   <property name="text">
    <string>11</string>
   </property>
  </action>
  <action name="action12">
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/icons/res/Bitmap11.bmp</normaloff>:/icons/res/Bitmap11.bmp</iconset>
   </property>
   <property name="text">
    <string>12</string>
   </property>
  </action>
  <action name="action13">
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/icons/res/Bitmap12.bmp</normaloff>:/icons/res/Bitmap12.bmp</iconset>
   </property>
   <property name="text">
    <string>13</string>
   </property>
  </action>
  <action name="action14">
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/icons/res/Bitmap13.bmp</normaloff>:/icons/res/Bitmap13.bmp</iconset>
   </property>
   <property name="text">
    <string>14</string>
   </property>
  </action>
  <action name="action15">
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/icons/res/Bitmap14.bmp</normaloff>:/icons/res/Bitmap14.bmp</iconset>
   </property>
   <property name="text">
    <string>15</string>
   </property>
  </action>
  <action name="action16">
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/icons/res/Bitmap15.bmp</normaloff>:/icons/res/Bitmap15.bmp</iconset>
   </property>
   <property name="text">
    <string>16</string>
   </property>
  </action>
  <action name="action17">
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/icons/res/Bitmap16.bmp</normaloff>:/icons/res/Bitmap16.bmp</iconset>
   </property>
   <property name="text">
    <string>17</string>
   </property>
  </action>
  <action name="action18">
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/icons/res/Bitmap41.bmp</normaloff>:/icons/res/Bitmap41.bmp</iconset>
   </property>
   <property name="text">
    <string>18</string>
   </property>
  </action>
  <action name="action19">
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/icons/res/Bitmap43.bmp</normaloff>:/icons/res/Bitmap43.bmp</iconset>
   </property>
   <property name="text">
    <string>19</string>
   </property>
  </action>
  <action name="action20">
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/icons/res/Bitmap17.bmp</normaloff>:/icons/res/Bitmap17.bmp</iconset>
   </property>
   <property name="text">
    <string>20</string>
   </property>
  </action>
  <action name="action21">
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/icons/res/Bitmap18.bmp</normaloff>:/icons/res/Bitmap18.bmp</iconset>
   </property>
   <property name="text">
    <string>21</string>
   </property>
  </action>
  <action name="action22">
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/icons/res/Bitmap19.bmp</normaloff>:/icons/res/Bitmap19.bmp</iconset>
   </property>
   <property name="text">
    <string>22</string>
   </property>
  </action>
  <action name="action23">
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/icons/res/Bitmap20.bmp</normaloff>:/icons/res/Bitmap20.bmp</iconset>
   </property>
   <property name="text">
    <string>23</string>
   </property>
  </action>
  <action name="action24">
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/icons/res/Bitmap21.bmp</normaloff>:/icons/res/Bitmap21.bmp</iconset>
   </property>
   <property name="text">
    <string>24</string>
   </property>
  </action>
  <action name="action25">
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/icons/res/Bitmap22.bmp</normaloff>:/icons/res/Bitmap22.bmp</iconset>
   </property>
   <property name="text">
    <string>25</string>
   </property>
  </action>
  <action name="action26">
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/icons/res/Bitmap23.bmp</normaloff>:/icons/res/Bitmap23.bmp</iconset>
   </property>
   <property name="text">
    <string>26</string>
   </property>
  </action>
  <action name="action27">
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/icons/res/Bitmap24.bmp</normaloff>:/icons/res/Bitmap24.bmp</iconset>
   </property>
   <property name="text">
    <string>27</string>
   </property>
  </action>
  <action name="action28">
   <property name="text">
    <string>28</string>
   </property>
  </action>
  <action name="action29">
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/icons/notOpen/empty/res/Bitmap26.bmp</normaloff>:/icons/notOpen/empty/res/Bitmap26.bmp</iconset>
   </property>
   <property name="text">
    <string>29</string>
   </property>
  </action>
  <action name="action30">
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/icons/notOpen/empty/res/Bitmap27.bmp</normaloff>:/icons/notOpen/empty/res/Bitmap27.bmp</iconset>
   </property>
   <property name="text">
    <string>30</string>
   </property>
  </action>
  <action name="action31">
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/icons/notOpen/empty/res/Bitmap28.bmp</normaloff>:/icons/notOpen/empty/res/Bitmap28.bmp</iconset>
   </property>
   <property name="text">
    <string>31</string>
   </property>
  </action>
  <action name="action32">
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/icons/notOpen/empty/res/Bitmap29.bmp</normaloff>:/icons/notOpen/empty/res/Bitmap29.bmp</iconset>
   </property>
   <property name="text">
    <string>32</string>
   </property>
  </action>
  <action name="action33">
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/icons/notOpen/empty/res/Bitmap30.bmp</normaloff>:/icons/notOpen/empty/res/Bitmap30.bmp</iconset>
   </property>
   <property name="text">
    <string>33</string>
   </property>
  </action>
  <action name="action34">
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/icons/notOpen/empty/res/Bitmap31.bmp</normaloff>:/icons/notOpen/empty/res/Bitmap31.bmp</iconset>
   </property>
   <property name="text">
    <string>34</string>
   </property>
  </action>
  <action name="action35">
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/icons/notOpen/empty/res/Bitmap32.bmp</normaloff>:/icons/notOpen/empty/res/Bitmap32.bmp</iconset>
   </property>
   <property name="text">
    <string>35</string>
   </property>
  </action>
  <action name="action36">
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/icons/notOpen/empty/res/Bitmap33.bmp</normaloff>:/icons/notOpen/empty/res/Bitmap33.bmp</iconset>
   </property>
   <property name="text">
    <string>36</string>
   </property>
  </action>
  <action name="action37">
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/icons/notOpen/empty/res/Bitmap34.bmp</normaloff>:/icons/notOpen/empty/res/Bitmap34.bmp</iconset>
   </property>
   <property name="text">
    <string>37</string>
   </property>
  </action>
  <action name="action38">
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/icons/notOpen/empty/res/Bitmap35.bmp</normaloff>:/icons/notOpen/empty/res/Bitmap35.bmp</iconset>
   </property>
   <property name="text">
    <string>38</string>
   </property>
  </action>
  <action name="action39">
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/icons/res/Bitmap25.bmp</normaloff>:/icons/res/Bitmap25.bmp</iconset>
   </property>
   <property name="text">
    <string>Открыть ключи</string>
   </property>
   <property name="toolTip">
    <string>Открыть ключи</string>
   </property>
   <property name="iconVisibleInMenu">
    <bool>false</bool>
   </property>
  </action>
  <action name="action40">
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/icons/res/Bitmap44.bmp</normaloff>:/icons/res/Bitmap44.bmp</iconset>
   </property>
   <property name="text">
    <string>40</string>
   </property>
  </action>
  <action name="action41">
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/icons/res/Bitmap36.bmp</normaloff>:/icons/res/Bitmap36.bmp</iconset>
   </property>
   <property name="text">
    <string>41</string>
   </property>
  </action>
  <action name="action42">
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/icons/res/Bitmap37.bmp</normaloff>:/icons/res/Bitmap37.bmp</iconset>
   </property>
   <property name="text">
    <string>42</string>
   </property>
  </action>
  <action name="action43">
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/icons/res/Bitmap38.bmp</normaloff>:/icons/res/Bitmap38.bmp</iconset>
   </property>
   <property name="text">
    <string>43</string>
   </property>
  </action>
  <action name="action44">
   <property name="icon">
    <iconset resource="res.qrc">
     <normaloff>:/icons/res/Bitmap39.bmp</normaloff>:/icons/res/Bitmap39.bmp</iconset>
   </property>
   <property name="text">
    <string>44</string>
   </property>
  </action>
  <action name="action">
   <property name="text">
    <string>Каскадом</string>
   </property>
  </action>
  <action name="action_3">
   <property name="text">
    <string>Таблицей</string>
   </property>
  </action>
  <action name="action_5">
   <property name="text">
    <string>Следующее</string>
   </property>
  </action>
  <action name="action_6">
   <property name="text">
    <string>Предыдущее</string>
   </property>
  </action>
  <action name="action_4">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Найти симптом</string>
   </property>
  </action>
  <action name="action_7">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Найти из текущего симптома</string>
   </property>
  </action>
  <action name="action_8">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="text">
    <string>Расширенный поиск...</string>
   </property>
  </action>
 </widget>
 <resources>
  <include location="res.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>action_2</sender>
   <signal>triggered()</signal>
   <receiver>MainWindow</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>501</x>
     <y>288</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
